airbyte-webapp/scripts/classname-serializer.js

```diff
@@ -0,0 +1,32 @@
+import { prettyDOM } from "@testing-library/react";
+
+/**
+ * Traverse a tree of nodes and replace all class names with
+ * the count of classnames instead, e.g. "<3 classnames>"
+ */
+const traverseAndRedactClasses = (node) => {
+  if (node.className && typeof node.className === "string") {
+    node.className = `<${node.className.split(" ").length} classnames>`;
```

Comment:
Is it important to have an accurate number of classes? `node.classList.length` could be more reliable for that, in case e.g. there are two consecutive spaces in a class name, `split(" ").length` would count an extra class.

If the length of classnames is not even relevant, perhaps we could just replace it with `class="<removed-for-snapshot-test">`

Reply:
It's not really important, I just tried to leave something. But as you said there's no real relevancy in that, so I'm going to change to just remove it altogether.